Unity-Technologies/ScriptableRenderPipeline
# Lw/fix template

### Purpose of this PR
 Fixed null reference in LWRP template.

---
### Release Notes
 Added

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=lw%2Ffix-template&automation-tools_branch=master&unity_branch=trunk

**Manual Tests**: What did you do?
- [x ] C# and shader warnings (supress shader cache to see them)
- Other: 

**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None

**Halo Effect**: None

---
### Comments to reviewers
Notes for the reviewers you have assigned.
